operator/scutils.py

```diff
@@ -0,0 +1,18 @@
+from string import Template
```

Comment:
For uniformity, use the Jinja2 template itself similar to other templates

Reply:
Done